Bottom: 42923f25e016c8166f4ba450c5198326fefd9086
Top:    0783cc4aaa2258936dae2558dc1d06b888b517fa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 15:14:38 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
